src/QsCompiler/Transformations/Monomorphization.cs

```diff
@@ -37,6 +38,10 @@ private struct Response
 
         public static QsCompilation Apply(QsCompilation compilation)
         {
+            CallGraph callGraph = BuildCallGraph.Apply(compilation);
+
+            var cycles = callGraph.GetCallCycles();
```

Comment:
What's the purpose of having this here? It isn't used by the monomorphization so far.

Reply:
removed